2023-03-08 19:01:06,316 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 12:03:46,530 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 13:53:56,527 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 13:55:26,738 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 13:57:43,032 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 14:01:44,649 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 14:02:43,502 - INFO - NewPersonnel : Personnel has been found!
2023-03-09 14:10:28,673 - ERROR - NewPersonnel : Failed
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 35, in AddNewPersonnel
    assert self.driver.current_url == "https://www.notion.so/"
AssertionError
2023-03-09 14:22:44,725 - ERROR - NewPersonnel : Item 'Ken Adams' not found in table
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 35, in AddNewPersonnel
    assert self.driver.current_url == "test"
AssertionError
2023-03-14 20:12:58,978 - INFO - NewDepartment : Department has been found!
2023-03-14 20:20:06,494 - INFO - NewDepartment : Department has been found!
2023-03-14 20:20:40,677 - INFO - NewPersonnel : Personnel has been found!
2023-03-14 20:32:33,326 - INFO - NewDepartment : Department has been found!
2023-03-14 20:33:15,248 - INFO - NewPersonnel : Personnel has been found!
2023-03-14 20:37:52,285 - INFO - NewDepartment : Department has been found!
2023-03-14 20:41:29,836 - INFO - NewDepartment : Department has been found!
2023-03-15 13:53:58,828 - INFO - NewDepartment : Department has been found!
2023-03-15 14:04:13,120 - INFO - NewDepartment : Department has been found!
2023-03-16 10:40:59,187 - INFO - NewDepartment : Department has been found!
2023-03-16 15:58:27,550 - INFO - NewDepartment : Department has been found!
2023-03-16 20:59:14,426 - INFO - NewDepartment : Department has been found!
2023-03-16 21:04:02,954 - INFO - NewDepartment : Department has been found!
2023-03-16 21:11:17,796 - INFO - NewDepartment : Department has been found!
2023-03-16 21:20:39,751 - INFO - NewDepartment : Department has been found!
2023-03-16 21:32:31,794 - INFO - NewPersonnel : Personnel has been found!
2023-03-16 21:53:35,579 - INFO - NewPersonnel : Personnel has been found!
2023-03-16 21:53:35,582 - ERROR - NewPersonnel : Item 'gocee' not found in table
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 44, in AddNewPersonnel
    assert row is None
AssertionError
2023-03-16 21:57:48,671 - INFO - NewDepartment : Department has been found!
2023-03-16 21:57:48,673 - ERROR - NewDepartment : 'Ritech Department 747' was not added!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/DepartmentPage.py", line 47, in AddNewDepartment
    assert row is None
AssertionError
2023-03-16 21:59:08,719 - INFO - NewDepartment : Department has been found!
2023-03-16 22:00:56,874 - INFO - NewDepartment : Department has been found!
2023-03-16 22:00:56,878 - ERROR - NewDepartment : 'Ritech Department 753' was not added!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/DepartmentPage.py", line 48, in AddNewDepartment
    assert row is None
AssertionError
2023-03-16 22:03:18,485 - INFO - NewDepartment : Department has been found!
2023-03-16 22:08:53,788 - INFO - NewDepartment : Department has been found!
2023-03-16 22:12:18,423 - INFO - NewDepartment : Department has been found!
2023-03-16 22:12:18,428 - ERROR - NewDepartment : 'Ritech Department 589' was not added!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/DepartmentPage.py", line 48, in AddNewDepartment
    assert row is None
AssertionError
2023-03-16 22:14:55,868 - ERROR - NewDepartment : 'Ritech Department 998' was not added!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/DepartmentPage.py", line 47, in AddNewDepartment
    assert row is None
AssertionError
2023-03-16 22:28:24,409 - INFO - NewDepartment : Department has been found!
2023-03-16 23:01:31,357 - INFO - NewDepartment : Department 'Ritech Department 497' has been found!
2023-03-16 23:06:40,972 - INFO - NewDepartment : Department 'Ritech Department 212' has been found!
2023-03-16 23:07:56,023 - INFO - NewDepartment : Department 'Ritech Department 63' has been found!
2023-03-16 23:15:09,003 - ERROR - NewPersonnel : Personnel 'jzhig' was not found!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 37, in AddNewPersonnel
    assert row is not None
AssertionError
2023-03-16 23:17:14,408 - INFO - NewPersonnel : Personnel 'jsxpj' has been found!
2023-03-16 23:20:47,656 - INFO - NewPersonnel : Personnel 'irrdr' has been found!
2023-03-16 23:22:38,514 - ERROR - NewPersonnel : Personnel 'exstq' was not found!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 37, in AddNewPersonnel
    assert row is not None
AssertionError
2023-03-17 13:47:32,541 - INFO - NewDepartment : Department 'Ritech Department 369' has been found!
2023-03-17 13:48:08,976 - INFO - NewPersonnel : Personnel 'suxzs' has been found!
2023-03-18 11:37:15,768 - INFO - NewDepartment : Department 'Ritech Department 255' has been found!
2023-03-18 11:38:15,150 - INFO - NewDepartment : Department 'Ritech Department 855' has been found!
2023-03-18 12:06:04,115 - INFO - NewDepartment : Department 'Ritech Department 668' has been found!
2023-03-19 21:10:12,109 - INFO - NewPersonnel : Personnel 'crtap' has been found!
2023-03-20 14:27:44,114 - INFO - ScheduleManager : Status updated!
2023-03-20 14:36:17,705 - INFO - ScheduleManager : Status updated!
2023-03-20 14:38:05,693 - INFO - ScheduleManager : Status updated!
2023-03-20 14:43:43,732 - ERROR - ScheduleManager : Status not updated!
NoneType: None
2023-03-20 14:45:13,174 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 78, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-03-20 14:46:46,804 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 78, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-03-20 14:49:30,883 - INFO - ScheduleManager : Status updated!
2023-03-20 15:29:24,521 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 132, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 11:17:33,516 - INFO - NewPersonnel : Personnel 'kdsev' has been found!
2023-04-01 11:19:46,383 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 132, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 11:40:03,683 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 132, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 11:53:29,849 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 81, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-01 11:58:59,154 - INFO - ScheduleManager : Status updated!
2023-04-01 12:02:02,293 - INFO - ScheduleManager : Status updated!
2023-04-01 12:03:21,816 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 134, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 12:24:17,979 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 134, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 12:28:19,147 - INFO - ScheduleManager : Status updated!
2023-04-01 16:48:45,227 - INFO - NewDepartment : Department 'Ritech Department 960' has been found!
2023-04-01 16:49:19,995 - INFO - NewPersonnel : Personnel 'ahkko' has been found!
2023-04-01 16:50:23,521 - INFO - ScheduleManager : Status updated!
2023-04-01 16:50:50,780 - INFO - ScheduleManager : Status updated!
2023-04-01 16:54:27,619 - INFO - NewDepartment : Department 'Ritech Department 544' has been found!
2023-04-01 16:55:05,612 - ERROR - NewPersonnel : Personnel 'cgxcr' was not found!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/PersonnelPage.py", line 47, in AddNewPersonnel
    assert row is not None
AssertionError
2023-04-01 16:56:19,541 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 81, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-01 16:56:55,532 - INFO - ScheduleManager : Status updated!
2023-04-01 18:06:20,107 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 80, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-01 18:10:35,053 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 80, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-01 18:12:34,942 - INFO - ScheduleManager : Status updated!
2023-04-01 18:31:58,828 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 80, in NewSchedule
    assert tds[5].text != "Published to Everyone"
AssertionError
2023-04-01 18:41:47,382 - INFO - ScheduleManager : Status updated!
2023-04-01 20:00:48,661 - INFO - NewDepartment : Department 'Ritech Department 592' has been found!
2023-04-01 20:02:31,827 - INFO - NewPersonnel : Personnel 'xribt' has been found!
2023-04-01 20:03:41,440 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 80, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-01 20:04:21,091 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 132, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-01 20:31:12,860 - INFO - NewDepartment : Department 'Ritech Department 649' has been found!
2023-04-01 20:32:57,372 - INFO - ScheduleManager : Status updated!
2023-04-01 20:33:31,401 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 132, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 12:06:14,435 - INFO - NewDepartment : Department 'Ritech Department 800' has been found!
2023-04-02 12:06:48,311 - INFO - NewPersonnel : Personnel 'lkthc' has been found!
2023-04-02 12:09:07,977 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 131, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 12:43:23,205 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 12:44:02,866 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 13:04:23,655 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 13:06:38,625 - INFO - ScheduleManager : Status updated!
2023-04-02 13:10:40,188 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-02 13:15:35,371 - INFO - ScheduleManager : Status updated!
2023-04-02 20:22:31,287 - INFO - NewDepartment : Department 'Ritech Department 430' has been found!
2023-04-02 20:23:06,372 - INFO - NewPersonnel : Personnel 'qgymh' has been found!
2023-04-02 20:24:11,504 - INFO - ScheduleManager : Status updated!
2023-04-02 20:25:33,801 - INFO - ScheduleManager : Status updated!
2023-04-02 20:27:01,351 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-03 15:16:41,973 - INFO - ScheduleManager : Status updated!
2023-04-03 15:41:31,218 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 80, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-03 15:42:57,181 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 133, in NewAutoGeneratedSchedule
    assert tds[5].text == "Generation Complete"
AssertionError
2023-04-06 12:12:31,963 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 83, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 12:19:18,302 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 83, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 12:21:21,802 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 83, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 13:06:13,230 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 82, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 13:10:17,905 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 82, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 13:37:25,418 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 84, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 14:50:20,656 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 84, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-06 14:53:26,372 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 84, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-24 00:02:44,489 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 91, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-24 00:10:00,824 - INFO - ScheduleManager : Status updated!
2023-04-24 00:37:05,637 - INFO - ScheduleManager : Status updated!
2023-04-24 00:50:13,052 - INFO - ScheduleManager : Status updated!
2023-04-24 00:57:36,866 - INFO - NewDepartment : Department 'Ritech Department 439' has been found!
2023-04-24 00:58:06,796 - INFO - NewPersonnel : Personnel 'efcnl' has been found!
2023-04-24 00:58:52,813 - INFO - ScheduleManager : Status updated!
2023-04-24 01:02:17,708 - INFO - NewDepartment : Department 'Ritech Department 602' has been found!
2023-04-24 01:02:46,705 - INFO - NewPersonnel : Personnel 'qdrcg' has been found!
2023-04-24 01:03:32,325 - ERROR - ScheduleManager : Status not updated!
Traceback (most recent call last):
  File "/Users/vjosaxhaferi/PycharmProjects/LightningBolt_Setup/pageObjects/ScheduleManagerPage.py", line 89, in NewSchedule
    assert tds[5].text == "Published to Everyone"
AssertionError
2023-04-24 01:06:51,685 - INFO - NewDepartment : Department 'Ritech Department 102' has been found!
2023-04-24 01:07:20,900 - INFO - NewPersonnel : Personnel 'yhxcq' has been found!
2023-04-24 01:08:05,193 - INFO - ScheduleManager : Status updated!
2023-04-24 01:09:21,263 - INFO - ScheduleManager : Status updated!
